config/cosmos.webpack.config.js

```diff
@@ -44,6 +44,16 @@ module.exports = {
         test: /\.(jpg|jpeg|png|gif|svg|eot|otf|ttf|woff|woff2)$/,
         loader: 'file-loader',
       },
+      {
+        test: /\.css$/,
+        exclude: /node_modules/,
+        use: ['style-loader', 'css-loader'],
+      },
+      {
+        test: /\.css$/,
```

Comment:
so we exclude `node_modules` in the first one and then we include them in the second ? I dont understand

Reply:
:smile:  copy paste from a tutorial (they wanted to convey a point) ... , yes it's the same as doing just one:
```
{
  test: /\.css$/,
  use: ['style-loader', 'css-loader'],
},
```